### PR TITLE
disable optimization for solc compiling

### PR DESCRIPTION
**Pull request by lightsing**
fix https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/1302

### CODE REVIEW

**Comment by lispc** [comment]
Patch diff  can be deleted

File: testool/src/compiler.rs [hunk 1]
```diff
@@ -71,6 +72,139 @@ pub struct Compiler {
...
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct SourceLocation {
+    end: i32,
+    file: String,
+    start: i32,
+}
+
 impl Compiler {
```
Comment (leolara) [inline testool/src/compiler.rs:208]: Why is the compiler impl so far from the compiler struct, if this compiler depends now in the new structs, perhaps we should move struct Compiler down, right before impl Compiler?

**Review by leolara — CHANGES_REQUESTED** [review]
Great adding more abstraction to this, but I have one question.

File: testool/src/compiler.rs (hunk 1 above)
Comment (lightsing) [reply testool/src/compiler.rs]: solved

**Review by lispc — APPROVED** [review]

**Review by leolara — APPROVED** [review]

**Pull request merged**